mono/pong/Pong with C#.csproj

```
@@ -1,6 +1,7 @@
-<Project Sdk="Godot.NET.Sdk/4.0.0-dev5">
+<Project Sdk="Godot.NET.Sdk/4.1.1">
   <PropertyGroup>
-    <TargetFramework>net472</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
+    <EnableDynamicLoading>true</EnableDynamicLoading>
     <RootNamespace>Pong</RootNamespace>
   </PropertyGroup>
 </Project>
```

```suggestion
</Project>

```
Missing line termination